controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -35,7 +35,7 @@ import (
 	h "github.com/redhat-appstudio/integration-service/helpers"
 	"github.com/redhat-appstudio/integration-service/loader"
 	"github.com/redhat-appstudio/integration-service/tekton"
-	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
```

Comment:
Why do we need to update the import alias here?

Reply:
I did it to make the import standardized over our packages. In all other cases we were using tektonv1beta1. This also makes it easier to figure out which version of the library we are using when referencing objects.